core/utilities.ex

```diff
@@ -1,7 +1,14 @@
 defmodule UltraDark.Utilities do
-
+ 
   def sha_base16(input), do: :crypto.hash(:sha256, input) |> Base.encode16
 
+  # Concatintes the list items together to a string and hashes the block header with  keccak sha3 algorithm
+  def sha3_hashing(input) do
+    converted_input=Enum.join(input)
```

Comment:
Lets add a space between the = sign. Also, does this produce an output in bytes or is it a string? If its bytes, lets run it through Base.encode16 to convert it to a string

Reply:
Lets also change the function name to be something like `sha3_hash` or `sha3_base16` if you end up passing it through Base.encode16